README.md

```diff
@@ -109,14 +109,28 @@ Supersonic is available in the AUR and can be built either manually with `makepk
   - ``export PATH="/Users/<yourname>/go/bin:$PATH"``
 * install the ``fyne`` packaging tool ``go install fyne.io/fyne/v2/cmd/fyne@latest``
 * install Xcode command-line tools (``xcode-select --install``)
+
+#### Homebrew
 * install libmpv (``brew install mpv``)
 * install dylibbundler (``brew install dylibbundler``) - needed only the bundledeps step, see below
 
+#### Macports
+* install mpv with the libmpv (and optionally vapoursynth, for bundledeps) variants (``sudo port install mpv +libmpv +vapoursynth``)
```

Comment:
I'm happy to merge this as-is, but I'm curious if the bundledeps for Macports builds could be tweaked to not assume vapoursynth is a dependency. AFAIK it's only for video, so if Supersonic can be built against an MPV lib that doesn't pull in vapoursynth as a dependency, it would potentially be a smaller, leaner build. So if you wanted to experiment, it'd be much appreciated, but if not I'll get around to it eventually (and maybe switch to building with macports as the default)

Reply:
Oh, yeah I should have given you more context. It was added because the Homebrew-packaged mpv requires linking to libvapoursynth, and it wasn't picked up properly by `dylibbundler`, so I made the shell script and worked it into the build process for Mac